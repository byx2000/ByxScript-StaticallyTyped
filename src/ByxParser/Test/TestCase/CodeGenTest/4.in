/*
	0 i
	1 x
	2 j
	3 y
	4 :i
	5 :x
	6 :z
	7 :k
	8 :v1
	9 :d1
*/

function void main()
{
	int i = 100;
	double x = 3.14;
	int j = 50;
	double y = 6.28;
	
	{
		int i = j;
		double x = y;
		double z = x;
		int k = i;
	}
	
	{
		int v1 = i;
		double d1 = x;
	}
}