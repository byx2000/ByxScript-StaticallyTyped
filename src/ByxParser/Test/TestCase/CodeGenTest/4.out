1
10 0

0

iconst 100
istore 0
dconst 3.14
dstore 1
iconst 50
istore 2
dconst 6.28
dstore 3
iload 2
istore 4
dload 3
dstore 5
dload 5
dstore 6
iload 4
istore 7
iload 0
istore 8
dload 1
dstore 9
ret